### PR TITLE
#160452473  Swagger Api Documentation

### PR DESCRIPTION
**Pull request by hnobi**
#### What does this PR do?
Adds swagger API documentation 
#### How should this be manually tested?
- `npm run start` and then  browse the Url `http://localhost:3000/api-docs/`

#### Any background context you want to provide?
Nil

#### What are the relevant pivotal tracker stories?
[#160452473](<https://www.pivotaltracker.com/story/show/160452473>)

#### Screenshots (if appropriate)
screen shot 2018-09-25 at 4 02 03 pm
#### Questions: Nil

### CODE REVIEW

File: swagger.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
```
Comment (macphilips) [inline swagger.json]: Great job. I would suggest you use YAML file rather JSON as YAML is far more easy to write and read than JSON. You can use [swaggerhub](https://app.swaggerhub.com) editor to edit the YAML directly.

Comment (hnobi) [reply swagger.json]: Ok, thanks  I have will look into it now

**Comment by omedale** [comment]
@hnobi test is failing here

**Comment by omedale** [comment]
@hnobi 
This does not really work as expected
- Wrong url `https://localhost/api/v1/api/users`
- Wrong Details `TypeError: Failed to fetch`

screen shot 2018-09-26 at 10 31 49 pm

**Review by omedale — CHANGES_REQUESTED** [review]
Please ensure this feature is properly done, a lot of other APIs that will be created in the future depens on it

**Review by macphilips — COMMENTED** [review]
Setup looks good, but some fields need to updated to ensure the try it function works properly. Great job

File: swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+swagger: '2.0'
+info:
+  title: Odin-ah-backend
+  description: " A Social platform for the creative at heart."
+  version: v1
+host: 127.0.0.1:3001
+produces:
+- application/json
+schemes:
+- https
```
Comment (macphilips) [inline swagger.yaml:11]: change schema to `http`

Comment (Alameen688) [reply swagger.yaml:11]: Yes, although we can have both .

```
schemes:
  - http
  - https
```

**Comment by Alameen688** [comment]
Tested. Looks good to me.

**Review by omedale — APPROVED** [review]

File: swagger.yaml (hunk 2 above)
Comment (omedale) [reply swagger.yaml:11]: If we can have both, that would be great

**Pull request merged**